target_users.md

```diff
@@ -0,0 +1,32 @@
+# CodeFlare Stack Target Users
+
+[Cluster Admin](#cluster-administrator)
+
+[Data Scientist I](#data-scientist-i)
+
+[Data Scientist II](#data-scientist-ii)
+
+
+
+## Cluster Administrator
+
+* Quota Management
+* Gang-Scheduling for Distributed Compute
+* Job/Infrastructure Queuing
+
+I want to enable a team of data scientists to have self-serve, but limited, access to a shared pool of distributed compute resources such as GPUs for large scale machine learning model training jobs. If the existing pool of resources is insufficient, I want my cluster to scale up (to a defined quota) to meet my users’ needs and scale back down automatically when their jobs have completed. I want these features to be made available through simple installation of Operators via the OperatorHub UI. I also want the ability to see the current MCAD queue, active and requested resources on my clusters, and the progress of all current jobs visualized in a simple dashboard.
```

Comment:
Can we make the target users desires more generic so that they don't mention CodeFlare or OpenShift specific technology? Something like:

" ...I envision achieving these capabilities through the straightforward installation of generic modules accessible via a user-friendly interface. I also want the ability to monitor the current queue of pending tasks, the utilization of active resources, and the overall progress of ongoing jobs through a simplified dashboard."